packages/junon-io/server/commands/getsubstring.js

```diff
@@ -0,0 +1,35 @@
+const BaseCommand = require("./base_command")
+
+class GetSubstring extends BaseCommand {
+    getUsage() {
+        return [
+            "Returns a variable named $substring",
+            "/getsubstring [substring] [message]",

```

Comment:
something like /getnthword [index] [message]     (index is the correct programmer term for numbers that reference position of something)